tests/message-extensions/src/card.ts

```diff
@@ -0,0 +1,182 @@
+import { ActionSet, Card, Element, Image, OpenUrlAction, TextBlock } from '@microsoft/teams.cards';
+import { Account, Message, ThumbnailCard } from '@microsoft/teams.api';
+
+interface FormData {
+  title: string;
+  subTitle: string;
+  text: string;
+}
+
+export function createCard(data: FormData) {
+  const cardImageUrl =
+    'https://github.com/microsoft/teams-agent-accelerator-samples/raw/main/python/memory-sample-agent/docs/images/memory-thumbnail.png';
```

Comment:
nit: Do we want to host this file on teams.ts repo? I'm fine with either, but something to keep in mind if the image is ever deleted on agent accelerator. Right now, we don't yet have a dependency checking URLs for validity, but I'd like to add it.

Reply:
+1, another idea is to re-use a URL that adaptive cards sample use

Reply:
Which adaptive cards sample are you referring to? the one under `tests` use a url but it doesn't point to an image